### PR TITLE
update go sdk and extract webpack config

### PR DESCRIPTION
**Pull request by dasfmi**
This PR resolves the security risks (check notes below) by upgrading the Grafana go SDK, and removes the unwanted keys in **plugin.json** by upgrading Grafana ui and frontend tools.

Most of the changes are done by the Grafana upgrade commands, the noticeable changes are towards extracting the webpack config for query editor.

---
1- Problem: osv-scanner detected a critical severity issue

    Details: SEVERITY: CRITICAL in package [github.com/grafana/grafana-plugin-sdk-go](http://github.com/grafana/grafana-plugin-sdk-go), vulnerable to CVE-2024-8986

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 services:
   grafana:
     user: root
-    container_name: 'axiom-axiomgrafana-datasource'
+    container_name: 'axiomhq-axiom-datasource'
```
Comment (bahlo) [inline docker-compose.yaml:4]: Is this generated? The name looks funny

File: pkg/plugin/datasource.go [hunk 2]
```diff
@@ -119,6 +119,20 @@ type queryModel struct {
 
 func (d *Datasource) query(ctx context.Context, query concurrent.Query) backend.DataResponse {
 	logger := log.DefaultLogger.FromContext(ctx)
+	// log panic
+	defer func() {
+		if r := recover(); r != nil {
+			var ok bool
```
Comment (bahlo) [inline pkg/plugin/datasource.go:125]:
```suggestion
```

You don't need this as you `:=` the line below

File: pkg/plugin/datasource.go [hunk 3]
```diff
@@ -119,6 +119,20 @@ type queryModel struct {
...
+			err, ok := r.(error)
+			if !ok {
+				err = fmt.Errorf("pkg: %v", r)
+				logger.Error(err.Error())
```
Comment (bahlo) [inline pkg/plugin/datasource.go:129]: Missing an early return

File: docker-compose.yaml (hunk 1 above)
Comment (dasfmi) [reply docker-compose.yaml:4]: this is the name used in [Grafana marketplace](https://grafana.com/grafana/plugins/axiomhq-axiom-datasource/). Don't know how did that start but it seems to follow this patter: `$orgName-$pluginName-datasource` but I checked other plugins like splunk and honeycomb, they are named: `grafana-splunk-plugin` and `grafana-honeycomb-plugin`

File: pkg/plugin/datasource.go (hunk 2 above)
Comment (dasfmi) [reply pkg/plugin/datasource.go]: updated.

File: pkg/plugin/datasource.go (hunk 3 above)
Comment (dasfmi) [reply pkg/plugin/datasource.go:128]: done 👍

**Review by bahlo — APPROVED** [review]

**Pull request merged**